aspnetcore/includes/not-latest-version.md

```diff
@@ -0,0 +1,4 @@
+> [!NOTE]
+> This isn't the latest version of this article. To switch to the latest, use the ASP.NET Core version selector at the top of the Table of contents.  
```

Comment:
I wonder if because this "Table of contents" isn't a screen item, it could avoid the strange proper noun casing. Just lowercase it, perhaps?

```suggestion
> This isn't the latest version of this article. To switch to the latest, use the ASP.NET Core version selector at the top of the table of contents.  
```

Reply:
Add comments on recommended usage.  Put comments around the following text

Right after latest released version (M), add the following markup:
```
:::moniker range="< aspnetcore-M.0"

[!INCLUDE [not-latest-version](~/includes/not-latest-version.md)]

:::moniker-end
```
That's how we should be using it. Makes it easy to update no matter what we do with includes going forward.

Reply:
> Right after latest released version (M), add the following markup:

So put this markup at the end of the main file or at the beginning of the earlier-version include file?

Reply:
I think end of main file before the include links makes the most sense.

Reply:
No, I just saw your comment in the issue about putting it at the top of the article main file, so I'm changing the instructions in the issue to say that:

> The following markup should be added to top of the main article file (not to any of the previous-version include files). `M` represents the currently released version number:
>
> ```
> :::moniker range="< aspnetcore-M.0"
> [!INCLUDE [not-latest-version](~/includes/not-latest-version.md)]
> :::moniker-end
> ```

Where else should we put these instructions?

Reply:
> Where else should we put these instructions?

Just remind folks when we review PR's. Any .MD file that in a PR that doesn't have this include should have it.

cc @wadepickett

Reply:
Having it at the top makes it easy to inspect the file.